qiskit/dagcircuit/dagcircuit.py

```diff
@@ -1007,6 +1009,34 @@ def num_tensor_factors(self):
         return rx.number_weakly_connected_components(self._multi_graph)
 
     def __eq__(self, other):
+        """Check if two circuits are equal.
+        This will check .cablirations and .global_phrase and
```

Comment:
```suggestion
        This will check .calibrations and .global_phrase and
```

Reply:
Thanks! and yup I saw this spelling error on both the documentation